src/logic/calculate.js

```diff
@@ -0,0 +1,8 @@
+import operate from './operate';
+
+const calculate = (data, buttonName) => {
+  const { total, next } = data;
+  return operate(total, next, buttonName);
```

Comment:
- This return statement should be returning the content of the modified data object, meaning `total, next, operation`, since this is going to be imported by the calculate file, remember that not every button name is an operation, so when you are passing on just numbers you are just taking in buttonName but not modifying the data object, as you should so that the call to operate is always receiving the correct data. And with the current configuration operate is only recieving the button name, as we need to modify `next, total and operation` according to the value of the buttonName argument.